src/compiler/checker.ts

```diff
@@ -25128,4 +25128,8 @@ namespace ts {
                 return false;
         }
     }
+
+    function enquote(s: string): string {

```

Comment:
`quote` is a fine name in my opinion. It's not ambiguous in our codebase and English verbs nouns without a required prefix*.

*Usually.

Reply:
I'm not sure two uses justifies the existence of this function actually. Your call.